libcloudforensics/providers/gcp/internal/monitoring.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Google Cloud Monitoring functionality."""
+
+import datetime
+from typing import TYPE_CHECKING, Dict
+
+from libcloudforensics.providers.gcp.internal import common
+
+if TYPE_CHECKING:
+  import googleapiclient
+
+
+class GoogleCloudMonitoring:
+  """Class to call Google Monitoring APIs.
+
+  https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.timeSeries
+
+  Attributes:
+    project_id: Project name.
+    gcm_api_client: Client to interact with Monitoring APIs.
+  """
+  CLOUD_MONITORING_API_VERSION = 'v3'
+
+  def __init__(self, project_id: str) -> None:
+    """Initialize the GoogleCloudMonitoring object.
+
+    Args:
+      project_id (str): The name of the project.
+    """
+
+    self.gcm_api_client = None
+    self.project_id = project_id
+
+  def GcmApi(self) -> 'googleapiclient.discovery.Resource':
```

Comment:
Note: I believe the style guide says to capitalize acronyms, but in this case it's OK to leave as-is as the resulting function name would be `GCMAPI`

Reply:
This aligns with the other, existing modules (eg GcfApi)